Democracy doc update (#3257)
w3f/polkadot-wiki: docs/maintain/maintain-guides-democracy.md

:::

> On Polkadot Apps you can navigate to Governance -> Democracy tab to make a new proposal. In order to submit a

Reviewer: ```suggestion
On Polkadot Apps you can navigate to the Governance -> Democracy tab to make a new proposal. In order to submit a
```

proposal, you will need to submit what's called the preimage hash. The preimage hash is simply the
hash of the proposal to be enacted. The easiest way to get the preimage hash is by clicking on the
"Submit preimage" button and configuring the action that you are proposing.

> The example below is demonstrated on Kusama (the same procedure applies to Polkadot as well). 

Reviewer: ```suggestion
The example below demonstrates the creation of a proposal on Kusama (**the same procedure applies to Polkadot**). 
```
